### ISSUE TITLE
Typo in Questbook

### ISSUE THREAD

**Issue by KatrinkaRose** [Tismas/cave-factory#46]
Currently playing version 1.1.0

The crafting station quest "Time to get some tools" mispells "opposite" and should be "its" instead of the current "it's"

Even spelled correctly that phrase is somewhat awkward and I would suggest rewording it to something like:
Unlike the standard crafting table the crafting station holds the items left in its grid when you close the table.

![image](https://user-images.githubusercontent.com/90729629/149597855-a3c262f2-735b-4459-b439-ef5273bc79ed.png)

**Comment by ghost** [comment]
This is not really an issue. I do not think the developers would make an update just to fix a small typo. Most people should be able to fix these grammatical errors in their brain, so it doesn't really matter.

**Comment by Bradley19** [comment]
i didnt even notice that
